### DISCUSSION TITLE
希望作者把神网手机版做的更还原

### DISCUSSION THREAD

**Discussion by LT-UT** [share121/inter-knot#3306] [其他]
![DM_20240820184629_001](https://github.com/user-attachments/assets/a20ffb21-5a9b-4820-b0dd-8f4c5761da9c)
绝区零官网的图片

**Comment by Junjun280** [comment]
可以从该链接向作者提意见或网站bug哦！https://github.com/share121/inter-knot/issues<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >

**Comment by mztxy229** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >再搞个knock knock跟绳网绑定（）
